xml/System.Numerics/BigInteger.xml

```diff
@@ -11405,11 +11405,12 @@ The integer value `33022` can be exported in four different arrays:
         <Parameter Name="result" Type="System.Numerics.BigInteger" RefType="out" Index="2" FrameworkAlternate="net-7.0;net-8.0" />
       </Parameters>
       <Docs>
-        <param name="s">To be added.</param>
-        <param name="provider">To be added.</param>
-        <param name="result">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="s">The string to parse.</param>
+        <param name="provider">An object that provides culture-specific formatting information about <code data-dev-comment-type="paramref">s</code>.</param>
+        <param name="result">When this method returns, contains the result of successfully parsing <code data-dev-comment-type="paramref">s</code> or an undefined value on failure.</param>
+        <summary>Tries to parse a string into a value.</summary>
+        <returns>
+          <code data-dev-comment-type="langword">true</code> if <code data-dev-comment-type="paramref">s</code> was successfully parsed; otherwise, <code data-dev-comment-type="langword">false</code>.</returns>
```

Comment:
I'm intrigued why the references were added that way:

```suggestion
          <see langword="true" /> if  <paramref name="s" /> was successfully parsed; otherwise, <see langword="false" />.</returns>
```